src/__integrationtests__/Attestation.spec.ts

```diff
@@ -71,10 +71,9 @@ describe('When there is an attester, claimer and ctype drivers license', () => {
       content,
       claimer.getAddress()
     )
-    const request = (await RequestForAttestation.fromClaimAndIdentity(
-      claim,
-      claimer
-    )).message
+    const request = (
+      await RequestForAttestation.fromClaimAndIdentity(claim, claimer)
+    ).message
```

Comment:
Just a style thing:

```suggestion
    const { message: request } = 
      await RequestForAttestation.fromClaimAndIdentity(claim, claimer)
```

Does this look better?

Reply:
It does, thank you.

Reply:
Fixed in 4e9832e